wiki/Main_Page/pt-br.md

```diff
@@ -1,61 +1,87 @@
 
-[Perguntas Frequentes](/wiki/FAQ) • [Multijogador](/wiki/Multi) • [Modos de Jogo](/wiki/Game_mode): ([osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania))
+[Formatos de arquivo](/wiki/osu!_File_Formats): [.osz](/wiki/osu!_File_Formats/Osz_(file_format)) • [.osk](/wiki/osu!_File_Formats/Osk_(file_format)) • [.osr](/wiki/osu!_File_Formats/Osr_(file_format)) • [.osu](/wiki/osu!_File_Formats/Osu_(file_format)) • [.osb](/wiki/osu!_File_Formats/Osb_(file_format)) • [.db](/wiki/osu!_File_Formats/Db_(file_format))
 
 </div>
 <div class="wiki-main-page-panel">
 
+# Jogabilidade
 
-# [Editor de Beatmap](/wiki/Beatmap_Editor)
+[Modos de Jogo](/wiki/Game_Mode): [osu!](/wiki/Game_Mode/osu!) • [osu!taiko](/wiki/Game_Mode/osu!taiko) • [osu!catch](/wiki/Game_Mode/osu!catch) • [osu!mania](/wiki/Game_Mode/osu!mania)
 
-Seções: ([Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song Setup](/wiki/Beatmap_Editor/Song_Setup)) • Componentes: ([Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance Snap](/wiki/Beatmap_Editor/Distance_Snap)) • [Beatmapping](/wiki/Beatmapping) • [Dificuldades](/wiki/Difficulties) • [Hit Objects](/wiki/Hit_Objects) • [Técnicas de mapping](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
+[Beatmap](/wiki/Beatmaps) • [Hit object](/wiki/Hit_object) • [Mods](/wiki/Game_Modifiers) • [Pontuação](/wiki/Score) • [Replay](/wiki/Replay) • [Multi](/wiki/Multi)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# [Editor de beatmap](/wiki/Beatmap_Editor)
+
+Seções: [Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song setup](/wiki/Beatmap_Editor/Song_Setup)
 
-# Edição online e Aprovação
+Componentes: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Beat snap divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance snap](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB load](/wiki/Beatmap_Editor/SB_Load) • [Timelines](/wiki/Beatmap_Editor/Timelines)
 
-[Submissão](/wiki/Submission) • [Aprovação de Beatmaps](/wiki/Beatmap_ranking_procedure) • [Modding](/wiki/Modding) • [Ranking Criteria](/wiki/Ranking_Criteria): ([osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) •[osu!mania](/wiki/Ranking_Criteria/osu!mania))
+[Beatmapping](/wiki/Beatmapping) • [Dificuldades](/wiki/Difficulties) • [Técnicas de mapping](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# Envio de beatmap e avaliação
 
-# Mundo de osu!
+[Envio](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimento para ranqueado](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
```

Comment:
```suggestion
[Envio](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimento para ranqueamento](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
```
How about `Procedimento para ranqueamento` for this one? Or just, well, you know. `Procedimento para ranqueamento de beatmap` if we are to go 1:1 with English. I do think it's a mouthful, albeit correct.